infra/lib/log-groups-stack.ts

```diff
@@ -15,8 +20,17 @@ export class LogGroupsStack extends Stack {
   constructor(scope: Construct, id: string, props: LogGroupsStackProps) {
     super(scope, id, props)
 
+    const FINNISH_SSN_REGEX =
+      "((0[1-9]|[12][0-9]|3[01]))((0[1-9]|1[0-2]))(\\d{2})([-AaBbCcDdEeFfXxYyWwVvUu+])(\\d{3})([0-9A-Ya-y])"
```

Comment:
There is no ready rule for finnish SSN so we use our own regex

https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/protect-sensitive-log-data-types-pii.html